modules/cluster-logging-collector-external.adoc

```diff
@@ -22,6 +22,13 @@ endif::openshift-enterprise[]
 
 To configure the *out_forward* plug-in, Create a ConfigMap called `secure-forward` in the `openshift-logging` namespace. On the external aggregator, configure the Fluentd `secure-forward.conf` file. For more information on using the *out_forward* plug-in, see the link:https://docs.fluentd.org/output/forward[Fluentd documentation].
 
+[IMPORTANT]
+====
+Because of changes introduced with the new log forward feature, significant changes were required to the Fluentd *out_forward* support. In {product-title} 4.2, you configured `out_forward` by editing the `secure-forward.conf` section of the Fluentd ConfigMap. In {product-title} 4.3, you create a Custom Resource, as described below, to configure `out_forward`. As a result, when you update, the `secure-forward.conf` section of the Fluentd ConfigMap is removed. You could copy your current `secure-forward.conf` section before upgrading to help when creating the Custom Resource. 
```

Comment:
Changes introduced by the new log forward feature modified the support for *out_forward* starting with the {product-title} 4.3 release.   In {product-title} 4.3, you create a ConfigMap as described below, to configure `out_forward`; any updates to the `secure-forward.conf` section of the Fluentd ConfigMap is removed. You can copy your current `secure-forward.conf` section before upgrading to use when creating the `secure-forward` ConfigMap.  Additionally you can add any certificates required by your configuration to a secret named `secure-forward` that will be mounted to the Fluentd pods.

Reply:
This assumes I have the object names correct as defined in the PR